orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -52,6 +52,8 @@ def execution_session_run_forward(execution_session, onnx_model, device, gradien
         forward_inputs = C.OrtValueVector()
         forward_inputs.reserve(len(inputs))
         for input in inputs:
+            if not input.is_contiguous():
+                input = input.contiguous()
```

Comment:
Maybe adding a warning message here? This leads to a tensor copy, which makes thing slow